build.rs

```diff
@@ -155,6 +155,20 @@ fn main() {
 
     ccfg
         .flag("-Wall")
+        .flag("-Wno-pointer-sign")
+        .flag("-Wno-unused-but-set-variable")
+        .flag("-Wno-unused-function")
+        .flag("-Wno-unused-value")
+        .flag("-Wno-unused-variable")
+        .flag("-Wmissing-include-dirs")
+        .flag("-Wredundant-decls")
+        .flag("-Wpointer-arith")
+        .flag("-Wformat=2")
+        .flag("-Wundef")
+        .flag("-Wunreachable-code")
+        .flag("-Wstrict-prototypes")
+        .flag("-Wold-style-definition")
+        .flag("-Wwrite-strings")
```

Comment:
AFAIK, the rust crate does not allow to test if a flag works or does not [currently provide a way to implement such test](https://github.com/alexcrichton/gcc-rs/issues/48).

Do you prefer to remove the additional flags and wait that the rust crate implements an API to do so? Do you prefer to check first if those flags are supported by other compilers? if so which ones (clang or another one?)?